base_import_pdf_by_template_account/models/account_move.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2024 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    def _get_edi_decoder(self, file_data, new=False):
+        if file_data["type"] == "pdf":
+            return self.env[
```

Comment:
This mustn't return, but try to import, and if not, call super.

Reply:
This was made similar to `account_edi_ubl_cii` (https://github.com/odoo/odoo/blob/17.0/addons/account_edi_ubl_cii/models/account_move.py#L52) because `account` processes the response in a “_special_” way https://github.com/odoo/odoo/blob/17.0/addons/account/models/account_move.py#L3203 but **ok**, I change it to what you comment.